dynamicweb/settings/base.py

```diff
@@ -573,27 +573,57 @@ def int_env(val, default_value=0):
 if 'info@ungleich.ch' not in DCL_ERROR_EMAILS_TO_LIST:
     DCL_ERROR_EMAILS_TO_LIST.append('info@ungleich.ch')
 
-ENABLE_DEBUG_LOGGING = bool_env('ENABLE_DEBUG_LOGGING')
-
-if ENABLE_DEBUG_LOGGING:
+ENABLE_LOGGING = bool_env('ENABLE_LOGGING')
+MODULES_TO_LOG = env('MODULES_TO_LOG')
+LOG_LEVEL = env('LOG_LEVEL')
+
+if LOG_LEVEL is None:
+    LOG_LEVEL = 'DEBUG'
+
+if ENABLE_LOGGING:
+    loggers_dict = {}
+    handlers_dict = {}
+    if MODULES_TO_LOG is None:
+        # set MODULES_TO_LOG to django, if it is not set
+        MODULES_TO_LOG = 'django'
+    if ',' in MODULES_TO_LOG:
```

Comment:
I think we can remove this condition and the entire following `else` block, as
`'string'.split(',')` would also yield `['string']`